### PR TITLE
Fix #1521: chromatic is not working

### LINKED ISSUE #1521: Chromatic Is Not Working
# Bug Report
https://www.chromatic.com/

PRs should have Chromatic-related actions, but they don't. Chromatic reports that we haven't built anything with them for a year!

### PR DESCRIPTION
**Pull request by recondesigns**
# Description of changes

Added chromatic script to package.json, and refactored Chromatic YAML for Github workflow. 

[https://www.chromatic.com/docs/github-actions#support-for-codeactionscheckoutv2code-and-above](url)

# Issue Resolved
Fixes #1521 Chromatic is not working

## Screenshots/GIFs
_None_

### CODE REVIEW

**Comment by recondesigns** [comment]
Not sure how to check if this is working since I am on on a fork and no member permission. I assume this would need to be merged in, and then a change to a component/story is pushed up.

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -1,7 +1,14 @@
-name: ci
+# .github/workflows/chromatic.yml
 
+# Workflow name
+name: 'Chromatic'
```
Comment (kylemh) [inline .github/workflows/chromatic.yml:4]: keep the name `ci` so the checks are all grouped together.

**Review by kylemh — COMMENTED** [review]
worth a try!

**Comment by kylemh** [comment]
I hit the merge button to see if it runs for me. Maybe this is a situation where Chromatic doesn't work for collaborators?

**Comment by recondesigns** [comment]
Can you hit that update branch thing again, or do you have any other ideas? I am curious if this needs to be merged into the `main` branch for Github Actions to fully pick it up?

I can get Chromatic to create/publish a new build, but only via CLI. When I push up a small change to a component, it does not trigger the Chromatic CI. 

Some other things I noticed
-  Input component has some errors that do not prevent the build/publish to Chromatic, but does block the CI
- Removing the stories for Input and and Select allow the Chromatic script to complete without an error
- When I start the Chromatic strip via CLI, the Storybook stuff does appear in the actions (screenshot)

Screen Shot 2023-05-30 at 2 44 08 AM

**Comment by kylemh** [comment]
You've done the hard part! You've fixed Chromatic 🎉 

Now we need to fix the 3 components that are failing to render. You can debug by running Storybook's dev server on those 3 components. Lemme know if you're uninterested! I can take over from here.

File: package.json [hunk 2]
```diff
@@ -26,7 +26,8 @@
     "test:changes": "yarn test --changedSince=main --verbose=true",
     "test:changes:watch": "yarn test:changes --watch",
     "test:watch": "yarn test --watch",
-    "test:update-snaps": "yarn test -u"
+    "test:update-snaps": "yarn test -u",
+    "chromatic": "chromatic --exit-zero-on-changes skip"
```
Comment (kylemh) [inline package.json:30]: can you alphabetize the scripts?

**Comment by recondesigns** [comment]
> You've done the hard part! You've fixed Chromatic 🎉
> 
> Now we need to fix the 3 components that are failing to render. You can debug by running Storybook's dev server on those 3 components. Lemme know if you're uninterested! I can take over from here.

Yea, I'll jump in and knock those out, as well as alphabetizing the scripts.

**Comment by recondesigns** [comment]
- I added Formik forms around Input and Select stories, and now they are rendering.
- I ran Chromatic via CLI, and the tests passed (may want to look into the auto accept flag so nobody needs to manually accept changes to components in Chromatic)
- Two of the "codeclimate" jobs of the workflow failed, and I am not totally sure what they are, or why

**Review by kylemh — COMMENTED** [review]
Amazing work!

**Pull request merged**